src/Client/Updater.php

```diff
@@ -575,4 +525,70 @@ private function getUpdateStartTime(): \DateTimeImmutable
     {
         return (new \DateTimeImmutable())->setTimestamp($this->clock->getCurrentTime());
     }
+
+    /**
+     * Performs a delegation search for a target metadata object.
```

Comment:
I'd rephrase this a bit: maybe "Searches delegated roles for metadata concerning a specific target"?

Reply:
fixed